### DISCUSSION TITLE
建议将lobechat 部署在国内服务器还是国外服务器？

### DISCUSSION THREAD

**Discussion by pipuwong** [lobehub/lobehub#4955] [General Question] [answered]
有没有一些服务，需要用国外服务器才能用？
如果部署在国内，会有些功能或者服务，访问不了的问题出现吗？

**Comment by breakstring** [comment, marked as answer]
> 有没有一些服务，需要用国外服务器才能用？ 如果部署在国内，会有些功能或者服务，访问不了的问题出现吗？

默认所需要的 OpenAI 的 API 必须在海外服务器才好访问吧，所以如果你部署在国内那么至少你还得在国外弄一个代理/转发
